support-frontend/assets/helpers/render.js

```diff
@@ -3,6 +3,13 @@
 import ReactDOM from 'react-dom';
 import { logException } from 'helpers/logger';
 
+// Without this the build-time pre-rendering breaks, because fetch is undefined when running with node
+const safeFetch = (url: string, opts) => {
+  if (typeof fetch !== 'undefined') {
+    fetch(url, opts)
+  }
```

Comment:
can we have another alert that happens if fetch is not defined and we're on the client side? 😆 📈